### ISSUE TITLE
Query builder: incorrect generation of column name for mysql data source

### ISSUE THREAD

**Issue by sdalu** [grafana/grafana#51283]
When using mysql data source column name are generated without backticks which result in a mysql error when the column name contains an hyphen.

For example if a column is called cloud-base

What is generated
~~~
SELECT
  time AS "time",
  cloud-base AS "Base",
...
~~~

What should be generated
~~~
SELECT
  time AS "time",
  `cloud-base` AS "Base",
...
~~~

**Environment**:
- Grafana version: 8.5.4
- Data source type & version: mariadb
- OS Grafana is installed on: freebsd

**Comment by ivanahuckova** [comment]
I was able to reproduce this on https://play.grafana.org/goto/Segpefqnk?orgId=1. Just click on `Toggle test edit mode` button.

**Comment by zoltanbedi** [comment]
Good news! This is fixed in 9.4.0 https://github.com/grafana/grafana/pull/61135
